tests/rust_libs/tests/01-run.py

```diff
@@ -7,34 +7,25 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
+import json
 import sys
 from testrunner import run
 
 
-EXPECTED_HELP = (
-    'Command              Description',
-    '---------------------------------------',
-    'bufsize              Get the shell\'s buffer size',
-    'start_test           starts a test',
-    'end_test             ends a test',
-    'echo                 prints the input command',
-    'empty                print nothing on command',
-    'hello_world          Print a greeting',
-    'xfa_test1            xfa test command 1',
-    'xfa_test2            xfa test command 2',
-)
+EXPECTED_CMDS = {
+        'bufsize': 'Get the shell\'s buffer size',
+        'start_test': 'starts a test',
+        'end_test': 'ends a test',
+        'echo': 'prints the input command',
+        'empty': 'print nothing on command',
+        'periodic': 'periodically print command',
```

Comment:
This was not tested against before. The way it was implemented before it would skip over lines in-between the given list of output lines. The test is now a bit stricter in this regard.